sklearn/notebooks/hp_tuning_auto_mpg/auto_mpg_hp_tuning/train.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
```

Comment:
Crystal told me we only need to provide the notebook file. So, you might not even need tran.py and __init__.py as separate files in this PR.

Reply:
Will remove